composer.json

```diff
@@ -31,10 +31,14 @@
     "require-dev": {
         "silverstripe/documentation-lint": "^1",
         "silverstripe/recipe-testing": "^3",
+        "silverstripe/behat-extension": "^5.6.1",
         "silverstripe/frameworktest": "^1",
         "silverstripe/standards": "^1",
         "phpstan/extension-installer": "^1.3"
     },
+    "conflict": {
+        "silverstripe/versioned": "<2.4.4"
```

Comment:
This includes a fix for the workaround for phpunit - https://github.com/silverstripe/silverstripe-versioned/pull/520

Since it only for a unit test, and required API, I've put it in as a conflict rather than a requirement

Reply:
Updated